api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -35,14 +36,16 @@
   private static SecureRandom rnd = new SecureRandom();
 
   // This list must exactly match the scopes allowed via the GSuite Domain Admin page here:
-  // https://admin.google.com/AdminHome?chromeless=1#OGX:ManageOauthClients
+  // https://admin.google.com/fake-research-aou.org/AdminHome?chromeless=1#OGX:ManageOauthClients
```

Comment:
I set the particular SA referenced in this file to the new scopes on this page. We may also need to update other SA's and/or other domains for these scopes as well....